vhost-device-gpio/src/backend.rs

```diff
@@ -47,6 +44,8 @@ pub(crate) enum Error {
     CouldNotCreateBackend(crate::vhu_gpio::Error),
     #[error("Could not create daemon: {0}")]
     CouldNotCreateDaemon(vhost_user_backend::Error),
+    #[error("Failed to serve daemon: {0}")]
```

Comment:
Nitpick, not review: Failed to serve and failed to create sound kinda weird to me, could it be better if we just printed the `vhost_user_backend::Error` on its own? Or say `Fatal error: {0}` or similar.

Reply:
Got no opinion here. Will switch to "Fatal error:" then!